packages/fulfillment/src/migrations/Migration20240213165503.ts

```diff
@@ -0,0 +1,88 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240213165503 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('create table if not exists "fulfillment_address" ("id" text not null, "fulfillment_id" text null, "company" text null, "first_name" text null, "last_name" text null, "address_1" text null, "address_2" text null, "city" text null, "country_code" text null, "province" text null, "postal_code" text null, "phone" text null, "metadata" jsonb null, "created_at" timestamptz not null default now(), "updated_at" timestamptz not null default now(), "deleted_at" timestamptz null, constraint "fulfillment_address_pkey" primary key ("id"));');
```

Comment:
thought: One thing I would like us to do at a later point is format these, so they are easier to read. 

I pasted mine in ChatGPT and asked it to format it for me.